RUFAS/biophysical/animal/pen.py

```diff
@@ -1067,9 +1058,25 @@ def _attempt_formulation(
         else:
             user_defined_ration_dictionary = None
             tolerance = None
+        nutrient_standard = list(self.animals_in_pen.values())[0].nutrient_standard
+
+        if nutrient_standard is NutrientStandard.NASEM:
+            enteric_methane_list = []
+            urine_nitrogen_list = []
+            for animal in self.animals_in_pen.values():
+                enteric_methane_list.append(animal.digestive_system.enteric_methane_emission)
+                urine_nitrogen_list.append(animal.digestive_system.manure_excretion.urine_nitrogen)
+            pen_average_enteric_methane = sum(enteric_methane_list) / len(enteric_methane_list)
+            pen_average_urine_nitrogen = sum(urine_nitrogen_list) / len(urine_nitrogen_list)
```

Comment:
```suggestion
     If enteric_methane_list and urine_nitrogen_list:
            pen_average_enteric_methane = sum(enteric_methane_list) / len(enteric_methane_list)
            pen_average_urine_nitrogen = sum(urine_nitrogen_list) / len(urine_nitrogen_list)
     else:
            pen_average_enteric_methane = 0
            pen_average_urine_nitrogen = 0
```
Not sure what the right action should be for else, but I believe this could be easily determined by SMEs.

Reply:
This was done more to avoid the calculation when they aren't actually used! I'll take a second look to see if I should set the pen averages for enteric methane and urine nitrogen to `None` instead.